packages/terra-list/lib/_variables.scss

```diff
@@ -0,0 +1,12 @@
+// Variables
+// ==========================
+
+// Basic List Element Colors
+$terra-list-chevron-color: $terra-grey-30 !default;
+$terra-list-divider-color: $terra-grey-15 !default;
+
+// Selection Styles
+$terra-list-item-hover-background-color: $terra-blue-10 !default;
+$terra-list-item-selected-background-color: $terra-blue-10 !default;
+$terra-list-item-selected-divider-color: $terra-blue-70 !default;
+$terra-list-item-selected-hover-background-color: darken($terra-blue-10, 5%) !default;
```

Comment:
What is the calculated value of this function? Does this value exist in our current palette?
https://github.com/cerner/terra-core/blob/master/packages/terra-legacy-theme/src/_palette.scss#L45